.circleci/config.yml

```diff
@@ -58,6 +58,10 @@ jobs:
       - run:
           name: Run all tests
           command: cargo test
+      - persist_to_workspace:
+          root: .
+          paths:
+            - target/debug/node
```

Comment:
This will persist across the jobs and then be used in the deploy step. But do we want it to? Aren't there extra artifacts included for testing or special flags for production compilation?

Reply:
As discussed briefly in the retrospective, the reason we chose to use persisting (now debug mode) is the other option is to do a full Dockerfile build, which takes around 50 minutes. Probably because the docker building (as used on circleci) does a single core build, or is magnitudes slower.